File: JavaScript/ VowelCount.js
function getCount(str) {
  const baseVowels = ['a', 'e', 'i', 'o', 'u'];
  const result = str.split("").filter((letter) => baseVowels.includes(letter));
 
  return result.length;
}

console.assert(getCount('pedro') === 2, 'Erro no teste 1');
console.assert(getCount('abracadabra') === 5, 'Erro no teste 2');
console.assert(getCount('bys') === 0, 'Erro no teste 3');

File: Python/vowel _count.py
def get_count(sentence):
    vowels = 'aeiou'
    count_vowel = 0
  
    for letter in sentence:
        if letter in vowels:
            count_vowel += 1

    return count_vowel


print(get_count("abracadabra"))


# ou

def getCount(inputStr):
    return sum(inputStr.count(char) for char in ['a', 'e', 'i', 'o', 'u'])
